packages/native-stack/src/views/NativeStackView.native.tsx

```diff
@@ -433,4 +441,14 @@ const styles = StyleSheet.create({
   container: {
     flex: 1,
   },
+  scene: {
+    flex: 1,
+    flexDirection: 'column-reverse',
```

Comment:
Why don't you just reorder components in JSX instead? Then you could reuse the `container` style too

Reply:
@kacperkapusciak reordering that way works differently - i.e. it reflects the order of elements in the view hierarchy. in the view hierarchy, the element that comes later is positioned on top of the previous one. so here we place the custom header after the screen in JSX so that it's on top of the content (especially when absolutely positioned) - so that the touches on the header aren't blocked by the content.

a similar effect can be achieved by using `zIndex` style which does this reordering in the view hierarchy afaik, but often it doesn't seem to work properly (especially on Android) so we use this method. the same approach is also used in all other navigators which render a header.